src/ProjectTemplates/Web.ItemTemplates/content/ApiController/.template.config/localize/templatestrings.en.json

```diff
@@ -3,7 +3,9 @@
   "name": "API Controller",
   "description": "API Controller with or without read/write actions",
   "symbols/namespace/description": "namespace for the generated code",
+  "_symbols/namespace/description.comment": "Should this start w/ 'Namespace'?",
```

Comment:
Reviewers, please answer questions like this in this file and the other two templatestrings.en.json files. Other `templatestrings.*.json` files can be ignored since they're just like the templatestrings.en.json files except for the comment entries.